packages/editor/src/components/post-title/index.native.js

```diff
@@ -57,31 +58,45 @@ class PostTitle extends Component {
 		this.props.onSelect();
 	}
 
-	onPaste( { value, onChange, plainText, html } ) {
+	onPaste( { value, plainText, html } ) {
```

Comment:
The changes made to this handler are based on the web implementation:
https://github.com/WordPress/gutenberg/blob/ab04a95714af347cac9ee0f24a3e58abf66f27ea/packages/editor/src/components/post-title/index.js#L122-L162